ignition.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+const smallIgnition = `
+{
+  "ignition": {
+    "version": "2.2.0",
+    "config": {
+      "append": [
+        {
+	    "source": "data:text/plain;charset=utf-8;base64,{{ .MainConfig }}"
+        }
+      ]
+    }
+   },
+  "networkd": {
+    "units": [
+      {
+        "contents": "[Match]\nName=eth0\n\n[Network]\nAddress={{ .IfaceIP }}/30\nGateway={{ .Gateway }}\nDNS=1.1.1.1\nDNS=8.8.4.4",
```

Comment:
DNS needs to be also configurable or otherwise on on-prem this won't work, we will have to pass this to KVM operator and than pass as ENV to k8s-kvm container.

Reply:
I've added dns servers support